### PR TITLE
Update bootflow_2711.md

### PR DESCRIPTION
**Pull request by andrum99**
[Was: Document what happens when nRPIBOOT is enabled and asserted.]

Nope, that's rubbish but I've spotted a typo, so just fix that.

### CODE REVIEW

**Comment by lurch** [comment]
I've no idea if this level of detail is appropriate, so ping @timg236   (Ha, I bet you never saw that coming @andrum99 :rofl: )

File: hardware/raspberrypi/bootmodes/bootflow_2711.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ The boot flow for the ROM (first stage) is as follows:-
    * Check SPI EEPROM for second stage loader
       * Success - run second stage bootloader
       * Fail - continue
+* If OTP defines `nRPIBOOT` GPIO and `nRPIBOOT` is low
```
Comment (timg236) [inline hardware/raspberrypi/bootmodes/bootflow_2711.md:19]: This extra text is unecessary. The last step is a loop as described in the previous step. The nRPIBOOT GPIO was already checked at line 11

File: hardware/raspberrypi/bootmodes/bootflow_2711.md [hunk 2]
```diff
@@ -72,4 +74,4 @@ Please see the [bootloader configuration](../bcm2711_bootloader_config.md) page
       * Attempt to load firmware using USB device mode from the USB OTG port - see [usbboot](https://github.com/raspberrypi/usbboot). There is no timeout for `RPIBOOT` mode.
 
 ## Bootloader updates
-The bootloader may also be updated before the firmware is started if a `pieeprom.upd` file is found. Please see the [bootloader eeprom](../booteeprom.md) page for more information about bootloader updates.
+The bootloader may also be updated before the firmware is started if a `pieeprom.upd` file is found. Please see the [bootloader EEPROM](../booteeprom.md) page for more information about bootloader updates.
```
Comment (timg236) [inline hardware/raspberrypi/bootmodes/bootflow_2711.md:77]: ok

File: hardware/raspberrypi/bootmodes/bootflow_2711.md (hunk 1 above)
Comment (andrum99) [reply hardware/raspberrypi/bootmodes/bootflow_2711.md:19]: Apologies - I've once again not read the page properly. I will modify the PR so it just fixes the typo.

Comment (lurch) [reply hardware/raspberrypi/bootmodes/bootflow_2711.md:19]: Must be all that Scottish air :scotland: :mountain: :wink:

Comment (andrum99) [reply hardware/raspberrypi/bootmodes/bootflow_2711.md:19]: > Must be all that Scottish air

Bit racist Andrew 😋

Comment (timg236) [reply hardware/raspberrypi/bootmodes/bootflow_2711.md:19]: I tried to keep the pseudo code structure as close the ROM implementation as possible, the combination of checking the OTP config + GPIO state does require a strong cup of coffee though

**Pull request merged**